src/borg/remote.py

```diff
@@ -323,12 +323,15 @@ def borg_cmd(self, args, testing):
                 opts.append('--critical')
             else:
                 raise ValueError('log level missing, fix this code')
+        env_vars = []
+        if bool(os.environ.get('BORG_UNIQUE_HOSTNAME')):
+            env_vars.append('BORG_UNIQUE_HOSTNAME=yes')
```

Comment:
a while ago I changed semantics of env vars. in early borg and attic, non-present env vars meant False and any present ones meant True (even if e.g. FOO=False or FOO=0 was used - they would be evaluated as truish).

after my change, env vars are handles like yes() input. So, it is FOO=0 and FOO=1 or FOO=yes and FOO=no now.

So, I think it should be:
```
unique_hostname = os.environ.get('BORG_UNIQUE_HOSTNAME')
if unique_hostname is not None:
    env_vars.append('BORG_UNIQUE_HOSTNAME=%s' % unique_hostname)
```